assets/charts/components/dex/templates/00-common.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
```

Comment:
Can we split manifests into standard file names like `serviceaccount.yaml`, `configmap.yaml` etc?

Reply:
I also wouldn't use numbered prefixes for manifest names, as they are meaningless with Helm.

Reply:
cool